hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1277,6 +1277,23 @@ public function upgrade_1037() {
     
     return TRUE;
   }
+
+  /**
+   * Renames page title from  job contract end reason to contract end reasons
```

Comment:
I think this upgrader actually renames the option group title. It just happens that the option group title is used for the page title.